error-handling/src/test/java/com/palantir/remoting2/errors/SerializableErrorErrorDecoderTests.java

```diff
@@ -73,6 +74,37 @@ public void testSpecificException() {
     }
 
     @Test
+    public void testExceptionWithoutMessage() {
+        Exception exception = encodeAndDecode(new NullPointerException());
+        assertThat(exception, is(instanceOf(RemoteException.class)));
+        assertThat(exception.getMessage(), is("null"));
```

Comment:
was that the behavior before your change, too? maybe this is a bug and it should actually be an empty string.